packages/termux-auth/build.sh

```diff
@@ -0,0 +1,7 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/xeffyr/termux-auth
```

Comment:
My only concern is having this under an external repo. If it's accepted, maybe it should be moved to a Termux repo.

Reply:
Yes, such things should be in located in https://github.com/termux/. But only @fornwall can do this.